apps/graphql/fixtures/addressProperties.js

```diff
@@ -0,0 +1,37 @@
+const NATIONAL_HERITAGE = "NATIONAL_HERITAGE";
```

Comment:
I like how you decide to use `HERITAGE`. Maybe we can rename all the `monument` strings to this as well, what do you think? 

eg: `getRestrictionByTypeName(address, "Heritage")?.name;`